Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -606,6 +617,78 @@ private static void IsTouchOptimizedChangedCallback(DependencyObject d, Dependen
             rangeSelector.ArrangeForTouch();
         }
 
+        /// <summary>
+        /// Gets or sets the step value for range selector.
+        /// </summary>
+        /// <value>
+        /// The value for step.
+        /// </value>
+        public double StepValue
+        {
+            get
+            {
+                return (double)GetValue(StepValueProperty);
+            }
+
+            set
+            {
+                SetValue(StepValueProperty, value);
+            }
+        }
+
+        private static void StepValueChangedCallback(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var rangeSelector = d as RangeSelector;
+            if (rangeSelector == null)
+            {
+                return;
+            }
+
+            if (rangeSelector.StepValue != 0)
+            {
+                rangeSelector.ResetStepValues();
+            }
+        }
+
+        private void ResetStepValues()
+        {
+            if (StepValue != 0)
```

Comment:
Would be good to compare to 0.0, but better to probably pull out as a constant of 'DefaultStepFrequency' so it's clear what we're checking for (and use that in the property initialization too).